input/P15tSZ.yml

```diff
@@ -0,0 +1,12 @@
+mapper_class: MapperP15tSZ
+map_name: P15tSZ
+file_map: /mnt/extraspace/damonge/Datasets/P15tSZ/COM_CompMap_Compton-SZMap-milca-ymaps_2048_R2.00.fits
+file_noise: /mnt/extraspace/damonge/Datasets/P15tSZ/COM_CompMap_Compton-SZMap-milca-stddev_2048_R2.00.fits
```

Comment:
I don't think this attribute exists for the Planck tSZ map

Reply:
Removed!